src/test/resource/fake-user-data-1

```diff
@@ -0,0 +1 @@
+fake user data 1
```

Comment:
I think we should include the contents of this files inside the testing code itself instead of adding the files to the resource folder

Reply:
The command needs to read some data from the file system so I think it's necessary to keep these files in the resources.